source/content/guides/logs-pantheon/04-automate-log-downloads.md

```diff
@@ -71,7 +205,7 @@ You can run the script with parameters for reusability.
 
 1. Edit the 'collect-logs-rsync.sh' (you can choose to disable the AGGREGATE_NGINX):
   
-  ```bash{promptUser:user}
```

Comment:
Is this an unrelated fix? I'm fine with including it if it's generally beneficial.

Reply:
unrelated, just noticed it was used incorrectly as i was fixing other things on the page